src/utils/get-core-member-logins.ts

```diff
@@ -66,13 +65,7 @@ const getCoreTeamsAndLogins = async (codeOwners?: string[]) => {
   }
 
   const teamsAndLogins = await map(codeOwners, async team =>
-    octokit.teams
-      .listMembersInOrg({
-        org: context.repo.owner,
-        team_slug: team,
-        per_page: 100
-      })
```

Comment:
replacement for this, but with pagination, added in https://github.com/ExpediaGroup/github-helpers/pull/705